tests/Storage/Microsoft.Bot.Builder.Azure.Tests.Common/Microsoft.Bot.Builder.Azure.Tests.Common.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework Condition="'$(BuildTarget)' == 'netcoreapp31'">netcoreapp3.1</TargetFramework>
```

Comment:
Should we target netstandard 2.0 for this common project?

Reply:
great idea!